bessctl/measurement_utils.py

```diff
@@ -88,38 +88,31 @@ def rtt(self, percentile=None):
         Returns the dictionary of all RTT value if `percentile` is none.
         Otherwise returns the `percentile`th percentile RTT.
         """
-        if self.rtt is None:
-            return None
-        if percentile is None:
-            return self.rtt
+        if self.rtt is None or percentile is None:
+            self.rtt
```

Comment:
lost a `return` keyword (here and below)...

Reply:
Doh. Fixed